model/project_parser.go

```diff
@@ -574,6 +574,9 @@ func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, ide
 
 	// return intermediateProject even if we run into issues to show merge progress
 	for _, path := range intermediateProject.Include {
+		if opts == nil {
+			return nil, errors.Wrapf(err, LoadProjectError)
```

Comment:
Errors.Wrap here will return nil since error is nil. I'm not actually sure we should error here since that's kind of debilitating for users and really is more of an error on our side. Maybe logging at the Error or Critical level (which goes to ops-alerts) is better so that we're aware of the problem.

Reply:
critical logging